src/lib/notifications/backgroundActions.js

```diff
@@ -0,0 +1,93 @@
+import { Notifications } from 'react-native-notifications'
```

Comment:
this file also should be .native, as we won't have any notifications on web by now